llvm/test/CodeGen/RISCV/reduce-unnecessary-extension.ll

```diff
@@ -0,0 +1,119 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv64 -mattr=+zbb -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=RV64I %s
+
+@PL_reg_match_utf8 = external dso_local local_unnamed_addr global i8, align 1
```

Comment:
drop dso_local and loca_unnamed_addr everywhere

Reply:
I have dropped those in new commit.